ci-operator/step-registry/cucushift/upgrade/check/capability/cucushift-upgrade-check-capability-commands.sh

```diff
@@ -68,10 +68,11 @@ function get_caps_for_version_capset() {
     caps_string="baremetal marketplace openshift-samples Console Insights Storage CSISnapshot NodeTuning MachineAPI Build DeploymentConfig ImageRegistry"
     ;;
     "v4.15")
-    caps_string="baremetal marketplace openshift-samples Console Insights Storage CSISnapshot NodeTuning MachineAPI Build DeploymentConfig ImageRegistry OperatorLifecycleManager CloudCredential"
+    caps_string="baremetal marketplace openshift-samples Console Insights Storage CSISnapshot NodeTuning MachineAPI Build DeploymentConfig ImageRegistry OperatorLifecycleManager"
+    [[ "${CLUSTER_TYPE}" =~ ^packet.*$|^equinix.*$ ]] && caps_string="${caps_string} CloudCredential"
```

Comment:
Hmm, actually for v4.15, `CloudCredential` indeed become an optional cap, but does not matter with platform, right? For non-baremetal platform, we have to configure `CloudCredential` as additional cap, and enable it, so that move forwards the testing, right? So I think we should get the caps from job configurations, but not reply on `CLUSTER_TYPE`

Reply:
ah, you're right. Re-reviewing [OCPQE-18367](https://issues.redhat.com/browse/OCPQE-18367), looks like it was using 4.15.0-ec.3 for testing which doesn't include CCO cap yet. So the post check failed. It doesn't seem to be a CI issue.